.check.exs

```diff
@@ -0,0 +1,65 @@
+[
+  ## don't run tools concurrently
```

Comment:
Can you remove the comments about how to use it and instead add a link to its documentation?